core/src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -62,6 +62,14 @@ public static void setup() {
 
     private static String baseCurrencyCode = "BTC";
 
+    // Calls to isFiatCurrency and isCryptoCurrency are very frequent so we use a cache of the results.
+    // The main improvement was already achieved with using memoize for the source maps, but
+    // the caching still improves performance by about 20% for isCryptoCurrency and about 100%
```

Comment:
100% improvement sounds wrong, 10%?

Reply:
Wording is a bit bad... old version consumes double the time so new version has a 100% improvement. 1777 ms vs 3467 ms. I will rephrase is....